crates/shared/src/http_solver/model.rs

```diff
@@ -115,6 +115,8 @@ pub struct TokenInfoModel {
     pub normalize_priority: Option<u64>,
     #[serde_as(as = "Option<DecimalU256>")]
     pub internal_buffer: Option<U256>,
+    /// Is token in the external list containing only safe tokens
+    pub is_safe: bool,
```

Comment:
Let's be more explicit about what this means (so that one doesn't have to look up the comment describing it). Safe can mean so many things.

Maybe: `accepted_for_internalization` or the like

Reply:
Fixed.